src/LibGEOS.jl

```diff
@@ -191,8 +191,34 @@ mutable struct WKBWriter
     end
 end
 
+const _GLOBAL_CONTEXT = Ref{GEOSContext}()
+const _GLOBAL_CONTEXT_ALLOWED = Ref(false)
+
+function get_global_context()::GEOSContext
+    if _GLOBAL_CONTEXT_ALLOWED[]
+        _GLOBAL_CONTEXT[]
+    else
+        msg = """
+        LibGEOS global context disallowed, a GEOSContext must be passed explicitly.
```

Comment:
Can we maybe also add: "If you wish to enable it by default, set `LibGEOS.allow_global_context!(true)`\"?

Reply:
Resolved by https://github.com/JuliaGeo/LibGEOS.jl/pull/141/commits/ebe434ddb69d5bbcdb49b5c2442a75e54b6eb470 thanks!